packages/nextra-theme-docs/src/index.tsx

```diff
@@ -29,7 +29,7 @@ function useDirectoryInfo(pageMap: PageMapItem[]) {
     const fsPath = getFSRoute(asPath, locale)
     return normalizePages({
       list: pageMap,
-      locale,
+      locale: locale ? locale : 'en-US',
```

Comment:
`en-US` is always will be by default see https://github.com/shuding/nextra/pull/568

Reply:
Nextjs locale type is string | undefined here